app/src/main/java/org/openedx/app/AppRouter.kt

```diff
@@ -411,6 +408,12 @@ class AppRouter : AuthRouter, DiscoveryRouter, DashboardRouter, CourseRouter, Di
         replaceFragmentWithBackStack(fm, VideoQualityFragment.newInstance(videoQualityType.name))
     }
 
+    override fun navigateToDiscover(fm: FragmentManager) {
+        fm.beginTransaction()
+            .replace(R.id.container, MainFragment.newInstance("", "", "DISCOVER"))
+            .commit()
+    }
```

Comment:
Can we use some constant instead of `"DISCOVERY"`?

Reply:
done